_sources/canasta/getMediawikiSettings.php

```diff
@@ -21,6 +21,12 @@ public function __construct() {
 			false,
 			true
 		);
+		$this->addOption(
+			'version',
+			'',
```

Comment:
Please document this option - what does it do, and how is it different from `versions`?